vm_env/src/imports.rs

```diff
@@ -216,6 +217,19 @@ pub mod tx {
         BlockHeight(unsafe { anoma_tx_get_block_height() })
     }
 
+    /// Get time of the current block header as rfc3339
+    pub fn get_block_time() -> String {
```

Comment:
I think maybe we could wrap the string with `Rfc3339String` here instead of in `vm_env/src/ibc.rs`